packages/material-ui/src/Drawer/Drawer.test.js

```diff
@@ -41,183 +45,152 @@ describe('<Drawer />', () => {
   );
 
   describe('prop: variant=temporary', () => {
-    it('should render a Modal', () => {
-      const wrapper = mount(
-        <Drawer>
-          <div />
-        </Drawer>,
-      );
-      expect(wrapper.find(Modal).exists()).to.equal(true);
-    });
-
-    it('should render Slide > Paper inside the Modal', () => {
-      const wrapper = mount(
-        <Drawer open>
-          <div />
-        </Drawer>,
-      );
-      const modal = wrapper.find(Modal);
-
-      const slide = modal.find(Slide);
-      expect(slide.exists()).to.equal(true);
-
-      const paper = slide.find(Paper);
-      expect(paper.exists()).to.equal(true);
-      expect(paper.hasClass(classes.paper)).to.equal(true);
-    });
-
     describe('transitionDuration property', () => {
       const transitionDuration = {
         enter: 854,
         exit: 2967,
       };
 
-      it('should be passed to Slide', () => {
-        const wrapper = mount(
-          <Drawer open transitionDuration={transitionDuration}>
+      it('delay the slide transition to complete', () => {
+        const handleEntered = spy();
+        const { setProps } = render(
+          <Drawer
+            open={false}
+            transitionDuration={transitionDuration}
+            SlideProps={{ onEntered: handleEntered }}
+          >
             <div />
           </Drawer>,
         );
-        expect(wrapper.find(Slide).props().timeout).to.equal(transitionDuration);
-      });
 
-      it("should be passed to to Modal's BackdropTransitionDuration when open=true", () => {
-        const wrapper = mount(
-          <Drawer open transitionDuration={transitionDuration}>
-            <div />
-          </Drawer>,
-        );
-        expect(wrapper.find(Modal).props().BackdropProps.transitionDuration).to.equal(
-          transitionDuration,
-        );
-      });
-    });
+        setProps({ open: true });
 
-    it("should override Modal's BackdropTransitionDuration from prop when specified", () => {
-      const testDuration = 335;
-      const wrapper = mount(
-        <Drawer BackdropTransitionDuration={testDuration}>
```

Comment:
`BackdropTransitionDuration` is no longer implemented. I can check when it got removed.